### PR TITLE
D882719: n-1 limit change

### PR DESCRIPTION
**Pull request by apollo-crum**

### CODE REVIEW

File: pyaedt/emit_core/results/revision.py [hunk 1]
```diff
@@ -372,34 +372,34 @@ def notes(self, notes):
         self.emit_project._emit_api.save_project()
 
     @property
-    def max_n_to_1_instances(self):
+    def n_to_1_limit(self):
         """
         The maximum number of instances per band combination allowed to run for N to 1.
```
Comment (PipKat) [inline pyaedt/emit_core/results/revision.py:377]:
```suggestion
       Maximum number of instances per band combination allowed to run for N to 1.
```
We drop the article (The, An, A) at the beginning of the brief description..

File: pyaedt/emit_core/results/revision.py [hunk 2]
```diff
@@ -372,34 +372,34 @@ def notes(self, notes):
         self.emit_project._emit_api.save_project()
 
     @property
-    def max_n_to_1_instances(self):
+    def n_to_1_limit(self):
         """
-        The maximum number of instances per band combination allowed to run for N to 1.
+        Maximum number of inteference combinations to run per receiver for N to 1.
         A value of 0 disables N to 1 entirely.
```
Comment (PipKat) [inline pyaedt/emit_core/results/revision.py:378]:
```suggestion
        Maximum number of inteference combinations to run per receiver for N to 1.
        
        - A value of ``0`` disables N to 1 entirely.
```
I think having each of these as a bullet entry would help with comprehension.

File: pyaedt/emit_core/results/revision.py [hunk 3]
```diff
@@ -372,34 +372,34 @@ def notes(self, notes):
...
-        A value of -1 allows unlimited N to 1 instances.
+        A value of -1 allows unlimited N to 1 (N is set to the maximum).
```
Comment (PipKat) [inline pyaedt/emit_core/results/revision.py:379]:
```suggestion
        - A value of  ``-1`` allows unlimited N to 1. (N is set to the maximum.)
```

File: pyaedt/emit_core/results/revision.py [hunk 4]
```diff
@@ -372,34 +372,35 @@ def notes(self, notes):
         self.emit_project._emit_api.save_project()
 
     @property
-    def max_n_to_1_instances(self):
+    def n_to_1_limit(self):
         """
-        The maximum number of instances per band combination allowed to run for N to 1.
-        A value of 0 disables N to 1 entirely.
-        A value of -1 allows unlimited N to 1 instances.
+        Maximum number of inteference combinations to run per receiver for N to 1.
```
Comment (myoung301) [inline pyaedt/emit_core/results/revision.py:377]: @apollo-crum This was caught by precommit:

```suggestion
        Maximum number of interference combinations to run per receiver for N to 1.
```

**Review by myoung301 — APPROVED** [review]

**Review by MaxJPRey — APPROVED** [review]
LGTM.

**Pull request merged**